packages/aws-cdk-lib/aws-rds/test/cluster.test.ts

```diff
@@ -4391,6 +4392,107 @@ describe('cluster', () => {
       expect(() => cluster.grantDataApiAccess(role)).toThrow('Cannot grant Data API access when the Data API is disabled');
     });
   });
+
+  describe('manageMasterUserPassword prop', () => {
+    test('manageMasterUserPassword cfn property true when associated DB cluster prop is true', () => {
+      // Given
+      const stack = testStack();
+      const vpc = new ec2.Vpc(stack, 'VPC');
+
+      // WHEN
+      new DatabaseCluster(stack, 'Database', {
+        engine: DatabaseClusterEngine.auroraPostgres({ version: AuroraPostgresEngineVersion.VER_16_1 }),
+        manageMasterUserPassword: true,
+        vpc,
+        writer: ClusterInstance.serverlessV2('writer'),
+      });
+
+      // THEN
+      Template.fromStack(stack).hasResourceProperties('AWS::RDS::DBCluster', {
+        ManageMasterUserPassword: true,
+      });
+
+    });
+
+    test('throw error for setting `manageMasterUserPassword` to true while `credentials` props excludeCharacters is defined', () => {
```

Comment:
We could unify these test cases:
```ts
test.each([
  ['excludeCharacters', { excludeCharacters: '1234' }],
  ['secret', { secret: new sm.Secret(stack, 'secret') }],
])(''throw error for setting `manageMasterUserPassword` to true while `credentials` props %s is defined', (_, credentials) => { ... }

Reply:
> We could unify these test cases:
> 
> ```ts
> test.each([
>   ['excludeCharacters', { excludeCharacters: '1234' }],
>   ['secret', { secret: new sm.Secret(stack, 'secret') }],
> ])('throw error for setting `manageMasterUserPassword` to true while `credentials.%s` is defined', (_, credentials) => { ... }
> ```

Sorry about the wait! It took a bit to work through how I could define an external stack and still keep resources unique.